CHANGES.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 2.1.2
```

Comment:
Let's follow the https://semver.org/ and bump the minor version for the new features like this.
```suggestion
## 2.2.0
```